html2text/__init__.py

```diff
@@ -376,15 +381,38 @@ def handle_tag(self, tag, attrs, start):
                 self.blockquote -= 1
                 self.p()
 
+        def no_preceding_space(self):
```

Comment:
A small docstring would be great, although it's not necessary. The function name speaks by itself.
Good function naming